File: js_workspace/day13/Bird.js
/*
모든 새들의 어버이인 그냥 새를 정의한다 
*/
class Bird{
    constructor(color, age, name){
        //색상 및 나이는 모든 새들이 공통적으로 가져야할 변수로 정의
        this.color=color;
        this.age=age;         
        this.name=name;
        console.log("Im a bird");
    }
}

File: java_workspace/day4/Bird.java
/*
모든 새들의 최상위 객체인 그냥 새를 정의한다
*/
class Bird{
	/*최상위 객체일 수록 */
	boolean hasWing=true; //모든 새는 날개가 있으므로..
	String color;

	public Bird(String color){
		this.color=color;
		System.out.println("난 부모 새");
	}

}
